### DISCUSSION TITLE
Edge properties of multilayer materials

### DISCUSSION THREAD

**Discussion by ranyang123** [neperfepx/neper#742] [Q&A] [answered]
When I generated multi-layer materials and mesh using the following command, and imported them into Abaqus, I found that the edges of each layer are shared by two layers, so there are two properties. How can I solve this problem?
`neper -T -domain 'square(10,3)' -n "from_morpho::msfile(grainrefine)" -morpho "lamellar(w=1,pos=start,v=(0,1))::voronoi" -dim 2 -o d1
neper -M d1.tess -elttype hex -rcl 0.5 -format inp`
`grainrefine
1  10
2  10
3  10`
![image](https://github.com/neperfepx/neper/assets/143670982/bd08517d-674f-4473-ade6-71ec531ee61b)

**Comment by rquey** [comment, marked as answer]
Not possible to do in the current version.

I would generate the three slices independently using Neper and paste their meshes externally.
